### PR TITLE
Fix

### PR DESCRIPTION
**Pull request by lyssieth**
Basically just ran the formatter + fixed some warnings. If anything sucks, please yeet.

Specifics:
- made shit align basically everywhere
- - align on colon
- used tabs everywhere I could
- idk what else

### CODE REVIEW

**Comment by mininmobile** [comment]
can you just keep 0fb37cd and c696c30 and undo everything else pretty much :)

**Comment by lyssieth** [comment]
Undo the warnings in index.js too?

File: index.js [hunk 1]
```diff
@@ -142,26 +142,26 @@ function formatSettings(settings) {
...
-			     "        /* list of options and modifications in the form of variables */\n" +
-			     "    }\n" +
-			     "}\n\n" +
-			     "/* your import link would then be `https://customa.gitlab.io/Customa-Discord/Category/module.m.css\` */";
+				"Category {\n" +
+				"    module.m.css {\n" +
+				"        /* list of options and modifications in the form of variables */\n" +
+				"    }\n" +
+				"}\n\n" +
+				"/* your import link would then be `https://customa.gitlab.io/Customa-Discord/Category/module.m.css\` */";
```
Comment (mininmobile) [inline index.js:155]: align using spaces not tabs pls

File: index.js [hunk 2]
```diff
@@ -142,26 +142,26 @@ function formatSettings(settings) {
...
-			     "        list of options and modifications in the form of variables\n" +
-			     "    }\n" +
-			     "}\n\n" +
-			     "your import link would then be `https://customa.gitlab.io/Customa-Discord/Category/module.m.css`\n";
+				"Category {\n" +
+				"    module.m.css {\n" +
+				"        list of options and modifications in the form of variables\n" +
+				"    }\n" +
+				"}\n\n" +
+				"your import link would then be `https://customa.gitlab.io/Customa-Discord/Category/module.m.css`\n";
```
Comment (mininmobile) [inline index.js:164]: align using spaces not tabs pls

File: src/app.js [hunk 3]
```diff
@@ -4,27 +4,27 @@ module.exports = exports = {
 		minor: 0,
 		patch: 0,
 	},
-	cli: {
+	cli    : {
```
Comment (mininmobile) [inline src/app.js]: the ones for cli, usage, and alias are just ugly

File: index.js [hunk 4]
```diff
@@ -38,7 +38,7 @@ let args = process.argv.splice(2);
 
 		if (app.cli.options[arg] !== undefined) {
 			app.cli.options[arg] = (args[i + 1] || "--").startsWith("--") ?
-				true : args[i + 1];
+								   true : args[i + 1];
```
Comment (mininmobile) [inline index.js]: align using spaces not tabs pls

File: index.js [hunk 5]
```diff
@@ -71,7 +71,7 @@ if (options.version) { // --version
 
 	// outputs "version: MAJOR.MINOR" (and ".PATCH" if >0)
 	console.log("version: " +
-		`${v.major}.${v.minor}${v.patch ? `.${v.patch}` : ""}`);
+				`${v.major}.${v.minor}${v.patch ? `.${v.patch}` : ""}`);
```
Comment (mininmobile) [inline index.js]: align using spaces not tabs pls

File: index.js [hunk 6]
```diff
@@ -109,24 +109,24 @@ function formatSettings(settings) {
 				if (settings.length > 0) {
 					// for each variable in module
 					settings.forEach((setting) =>
-						// append setting and default value
-						s += `--${setting}: ${module[setting]};\n`);
+										 // append setting and default value
+										 s += `--${setting}: ${module[setting]};\n`);
```
Comment (mininmobile) [inline index.js]: align using spaces not tabs pls

File: index.js [hunk 7]
```diff
@@ -109,24 +109,24 @@ function formatSettings(settings) {
...
 				} else {
 					// append module name
 					s += `    ${_module} {\n`;
 
 					// for each variable in module
 					settings.forEach((setting) =>
-						// append setting and default value
-						s += `        --${setting}: ${module[setting]};\n`);
+										 // append setting and default value
+										 s += `        --${setting}: ${module[setting]};\n`);
```
Comment (mininmobile) [inline index.js]: align using spaces not tabs pls

File: index.js [hunk 8]
```diff
@@ -147,21 +147,21 @@ function formatSettings(settings) {
 		if (options.markdown) {
 			s += "```css\n";
 			s += "/* this is how this channel is organized */\n" +
-				"Category {\n" +
-				"    module.m.css {\n" +
-				"        /* list of options and modifications in the form of variables */\n" +
-				"    }\n" +
-				"}\n\n" +
-				"/* your import link would then be `https://customa.gitlab.io/Customa-Discord/Category/module.m.css\` */";
+				 "Category {\n" +
```
Comment (mininmobile) [inline index.js]: why just one

File: index.js [hunk 9]
```diff
@@ -147,21 +147,21 @@ function formatSettings(settings) {
...
-				"        list of options and modifications in the form of variables\n" +
-				"    }\n" +
-				"}\n\n" +
-				"your import link would then be `https://customa.gitlab.io/Customa-Discord/Category/module.m.css`\n";
+				 "Category {\n" +
+				 "    module.m.css {\n" +
+				 "        list of options and modifications in the form of variables\n" +
+				 "    }\n" +
+				 "}\n\n" +
+				 "your import link would then be `https://customa.gitlab.io/Customa-Discord/Category/module.m.css`\n";
```
Comment (mininmobile) [inline index.js]: why just one

File: index.js [hunk 10]
```diff
@@ -109,24 +109,24 @@ function formatSettings(settings) {
...
 				}
 			} else {
 				if (settings.length === 0) {
 					// fallback if module does not contain any options
 					// append module name and stub block
 					s += options.markdown
-						? `    ${_module} { /* no variables */ }\n`
-						: `    ${_module} { no variables }\n`
+						 ? `    ${_module} { /* no variables */ }\n`
+						 : `    ${_module} { no variables }\n`
```
Comment (mininmobile) [inline index.js]: why just one

File: package.json [hunk 11]
```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "author": "",
+  "author": "TestNick / mininmobile",
```
Comment (mininmobile) [inline package.json]: just `mininmobile` pls n thank

**Review by mininmobile — CHANGES_REQUESTED** [review]
what :) and what :(

File: index.js (hunk 1 above)
Comment (lyssieth) [reply index.js:155]: you: use tabs
also you: ^^

Comment (lyssieth) [reply index.js:155]: also that would mean I have to format the entire thing with spaces; it be fucky like that

**Review by lyssieth — COMMENTED** [review]
nothing changed because nothing could be changed

File: index.js (hunk 10 above)
Comment (lyssieth) [reply index.js]: I believe to align with 4-tab lengths? not 100% sure on that one tbh

File: index.js (hunk 8 above)
Comment (lyssieth) [reply index.js]: same as other

File: index.js (hunk 9 above)
Comment (lyssieth) [reply index.js]: same as other

**Comment by mininmobile** [comment]
begone

**Pull request closed without merging**